### PR TITLE
Add support for namespace to the /api/notify endpoint

### PR DESCRIPTION
**Pull request by pypingou**
Signed-off-by: Pierre-Yves Chibon <pingou@pingoured.fr>

### CODE REVIEW

File: pkgdb2/api/extras.py [hunk 1]
```diff
@@ -262,11 +275,14 @@ def api_notify():
     :kwarg eol: Set to True if you want to include end of life
         distributions
     :kwarg format: Specify if the output if text or json.
+    :kwarg namespace: Specify if the namespace should be included in the
+        output (defaults to False).
```
Comment (puiterwijk) [inline pkgdb2/api/extras.py]: Why do you make this an option?
It's just another key in a dict, so it should be fine to just always add it, which also makes it easier to test for correctness.

Comment (pypingou) [inline pkgdb2/api/extras.py]: Because that would change the output and thus break tools relying on this format

Comment (puiterwijk) [inline pkgdb2/api/extras.py]: ok

**Comment by pypingou** [comment]
Thanks! :)

**Pull request merged**